src/app/modules/gb-export-module/data-table-components/gb-data-table-grid/gb-data-table-grid.component.html

```diff
@@ -5,22 +5,11 @@
 <ng-template #showDataTableGrid>
   <div class="grid-container">
     <p-table [value]="rows" [columns]="cols" [resizableColumns]="true" [customSort]="true">
-      <!-- header using individual cell headers -->
-      <ng-template pTemplate="header" let-columns>
-        <tr>
-          <th *ngFor="let col of columns" class="table-header" pResizableColumn
-              style="color: transparent">
-          <span>
-            {{col.header}}
-          </span>
-          </th>
-        </tr>
-      </ng-template>
-
       <!-- table body -->
       <ng-template pTemplate="body" let-row let-columns="columns">
-        <tr>
-          <td *ngFor="let col of columns" class="ui-resizable-column">
+        <!-- Header row -->
+        <tr *ngIf="row.isHeaderRow">
+          <th *ngFor="let col of columns" class="table-header">
```

Comment:
Maybe better to keep `pResizableColumn` option? Also, with many columns it's unreadable now (example below), maybe some min-width of columns would be useful? 
![image](https://user-images.githubusercontent.com/20908736/66118580-08360c80-e5d7-11e9-9a53-39f7c729856f.png)

Reply:
I now changed the data table to auto layout. That improves the rendering of tables with many columns. The problem with the resizable columns is that is has quite stange behaviour when resizing.